src/en/developer-layers-interfaces.md

```diff
@@ -193,9 +180,11 @@ And in the reactive class, we will receive the RelationBase object, and invoke
 the configure method:
 
 ```python
-@when('website.available')
-def configure_website(website):
-  website.configure(80)
+from charms.reactive import context
+
+@when('endpoint.website.joined')
+def configure_website():
+    context.endpoints.website.configure(80)
```

Comment:
Does this "configure" all the relationships of that website? If so, great :+1:, then the name really fits the functionality.